src/applications/static-pages/authenticated-homepage/components/App/App.js

```diff
@@ -0,0 +1,41 @@
+// Node modules.
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { selectShowAuthenticatedHomepage } from 'applications/static-pages/authenticated-homepage/selectors';
+import environment from 'platform/utilities/environment';
+
+export const App = ({ showAuthHomePage }) => {
+  if (environment.isProduction) {
```

Comment:
Added this check for peace of mind. I really don't want to affect the homepage in prod.